modules/react-components/src/components/card/selection-card.tsx

```diff
@@ -175,7 +175,8 @@ export const SelectionCard: FunctionComponent<SelectionCardPropsInterface> = (
         <Card
             id={ id }
             className={ classes }
-            onClick={ onClick }
+            onClick={ disabled ? (/*if disabled noop*/) => void 0 : onClick }
```

Comment:
Why do we need an empty function when disabled? Couldn't we do as following? 

`onClick={ disabled ? null : onClick }`

The above is a cleaner solution rather than having an empty function.